src/velero/tasks.yaml

```diff
@@ -104,3 +109,34 @@ tasks:
   - name: e2e-test
     actions:
       - description: "Run Velero E2E tests"
+
+  - name: update-crds
+    description: "Update Velero CRDs from upstream"
+    actions:
+      - description: "Download Velero CRDs from upstream"
+        cmd: |-
+          bash -c '
```

Comment:
We should be able to set the shell to bash in the task to avoid this wrapper. I think we do that somewhere else?